ml-agents-envs/mlagents_envs/side_channel/environment_parameters_channel.py

```diff
@@ -35,3 +48,86 @@ def set_float_parameter(self, key: str, value: float) -> None:
         msg.write_int32(self.EnvironmentDataTypes.FLOAT)
         msg.write_float32(value)
         super().queue_message_to_send(msg)
+
+    def set_sampler_parameters(
+        self, key: str, sampler_settings: ParameterRandomizationSettings
+    ) -> None:
+        """
+        Sets an environment parameter sampler.
+        :param key: The string identifier of the parameter.
+        :param sampler_settings: The sampler specific hyperparameters
+        """
+        if isinstance(sampler_settings, UniformSettings):
+            self.set_uniform_sampler_parameters(
+                key,
+                sampler_settings.min_value,
+                sampler_settings.max_value,
+                sampler_settings.seed,
+            )
+        elif isinstance(sampler_settings, GaussianSettings):
+            self.set_gaussian_sampler_parameters(
+                key,
+                sampler_settings.mean,
+                sampler_settings.st_dev,
+                sampler_settings.seed,
+            )
+        elif isinstance(sampler_settings, MultiRangeUniformSettings):
+            self.set_multirangeuniform_sampler_parameters(
+                key, sampler_settings.intervals, sampler_settings.seed
+            )
```

Comment:
Would add a final `else` that throws an error like "Invalid sampler configuration type".